content/pages/mua-summary.md

```diff
@@ -0,0 +1,68 @@
+Title: MUA Summary
+Status: hidden
+
+Anyone with a user account to a WiCS machine must sign our [Machine Usage Agreement]({filename}/pages/mua.md),
+summarized below. Those with guest accounts do not need to sign the MUA, but they are still held
+to the rules and regulations outlined in it.
+
+# Acceptable and Unacceptable Use #
+
+The WiCS machines are to be used for learning, class work, personal projects, and 
+use in accordance with [WiCS aims]({filename}/pages/about.md), that does not violate
```

Comment:
That does not violate any of WiCS' usage policies seems a bit redundant. Maybe

in accordance with WiCS' aims, the full MUA, university policy, and local law.